app/serializers/api/v1/user_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class API::V1::UserSerializer
+  include FastJsonapi::ObjectSerializer
+  attributes :email,
+             :name,
+             :organization_title
```

Comment:
Can we include their `employer` and serialize the `employer` with `title` instead? 

Mainly, i'm curious how this is handled in this gem for comparison to my experience with active model serializers.

Reply:
Sure -- should I have the FE send `employer_title` instead? You had `organization_name` in #72 , but I was worried that these different field names for request/response were going to cause confusion.

Reply:
This is what i was thinking:

```
{
 email: 'email here',
 name: 'name here',
 employer: {
  title: 'title here',
  other_attrs: 'here'
 }
}
```